contrib/zbm-sign.pl

```diff
@@ -56,9 +60,9 @@
   or die "Cannot open ZBM dir: $ZBM";
 
 if ($SignBackups) {
-  @EFIBins = grep { !/signed\.efi$/i and /\.efi/i } readdir $ZBM_dir;
+  @EFIBins = sort grep { !/signed\.efi$/i and /\.efi/i } readdir $ZBM_dir;
```

Comment:
Why sort?

Reply:
Glad you asked. The `sbverify` command modifies the efi files last modified timestamp resulting in refind showing as the default ZBM the last of the this array (last of the array is first in rEFInd list). This `sort` solution allows the user to controll the rEFInd order via naming.

Do you see an issue with using sort?

Reply:
Yes, this is not the place to hack around rEFInd sorting specifics. There is no need to sort unless this script depends on the order of the array, which it does not.